src/sql/src/plan/statement/ddl.rs

```diff
@@ -2779,6 +2787,21 @@ fn plan_sink(
     // VERSION defaults to 0
     let version = version.unwrap_or(0);
 
+    let partition_strategy = match partition_strategy.as_deref() {
```

Comment:
lowercase the `partition_strategy` option value to allow `V0` and `V1` too?

Reply:
Users are not allowed to touch this option so we don't need to be defensive here